src/screens/home/components/subreddits/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from "react";
+import Subreddit from "./Subreddit";
+
+const Subreddits = () => {
+
+    const [subreddits, setSubreddit] = useState([]);
+
+    useEffect(() => {
+        fetch("https://www.reddit.com/subreddits.json").then(
+          res => {
+            if (res.status !== 200) {
+              console.log("Warning: Something is wrong with the api.");
+              return;
+            }
+            res.json().then(data => {
+              if (data != null)
+                setSubreddit(data.data.children);
+            });
+          }
+        )
+      }, []);
+
+    return (
+        <div className="subreddits-main">
+            <h2>Subreddits</h2  >
+            {subreddits && subreddits.map(subreddit => <Subreddit key={subreddit.id} subreddit={subreddit.data}/>)}
```

Comment:
Deconstruct the map param: 

`{subreddits && subreddits.map(({id, data}) => <Subreddit key={id} subreddit={data}/>)}`